Dumplings/Rpc/VerboseTransactionInfo.cs

```diff
@@ -61,5 +71,21 @@ private ulong CalculateEquality(IEnumerable<Money> values)
 
             return equality;
         }
+
+        public bool IsWasabi2Cj()
+        {
+            var isNativeSegwitOnly = Inputs.All(x => x.PrevOutput.ScriptPubKey.IsScriptType(ScriptType.P2WPKH)) && Outputs.All(x => x.ScriptPubKey.IsScriptType(ScriptType.P2WPKH)); // Segwit only outputs.
```

Comment:
Can you optimize this function? First check what is computationally easy, like the number of inputs and outputs - return immediately if the conditions are not met.